src/requests/features.jl

```diff
@@ -1,6 +1,6 @@
 function get_signatures(b, tls, sigs, server, visited = nothing) end # Fallback
 
-function get_signatures(b::StaticLint.Binding, tls::StaticLint.Scope, sigs::Vector{SignatureInformation}, server, visited = Base.IdSet{StaticLint.Binding}())
+function get_signatures(b::StaticLint.Binding, tls::StaticLint.Scope, sigs::Vector{SignatureInformation}, server, visited = StaticLint.Binding[])
```

Comment:
What's the reason for switching to `Vector` ? 
Well, could I also ask why you used `Base.IdSet` before instead of usual `Set` ?

Reply:
Using IdSet was a mistake, I had been using them in SymbolServer and forgot it was quite costly